### ISSUE TITLE
[Bug]: mo replay wal cost too long[3m54] with just one table 40+ thousands sysbech data 

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6419]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):777b7ebbe34b8d1d3fe0733de2db588a4fc40c71
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

table DDL:
CREATE TABLE `sbtest1` (
`id` INT NOT NULL AUTO_INCREMENT,
`k` INT DEFAULT 0,
`c` CHAR(120) DEFAULT null,
`pad` CHAR(60) DEFAULT null,
PRIMARY KEY (`id`)
)

each record size is less than 200 byte, and the total number of table is just 46000+, but when restart the mo, replay wal need cost nearly 4 minutes:

2022/11/08 22:31:36.648041 +0800 INFO db/open.go:114 open-tae {"operation": "replay", "operand": "checkpoints", "cost": "66.205551ms", "checkpointed": "1667916295188831757-0"}
2022/11/08 22:34:41.617797 +0800 INFO db/open.go:122 open-tae {"operation": "replay", "operand": "wal", "cost": "3m4.969696482s"}
2022/11/08 22:41:39.591767 +0800 INFO db/open.go:114 open-tae {"operation": "replay", "operand": "checkpoints", "cost": "60.880249ms", "checkpointed": "1667918086681642104-0"}
2022/11/08 22:45:33.993330 +0800 INFO db/open.go:122 open-tae {"operation": "replay", "operand": "wal", "cost": "3m54.401514188s"}

**Comment by volgariver6** [comment]
plz @jiangxinmeng1 help investigate this problem, first find out what is the root cause. thx.

**Comment by jiangxinmeng1** [comment]
The read size in replay is too small. I'll change it to 2M.

**Comment by aressu1985** [comment]
Testing

**Comment by aressu1985** [comment]
MO can not started, now  this issue should be delayed to regress till #6526 is fixed

**Comment by aressu1985** [comment]
@jiangxinmeng1 pls confirm this issue to @volgariver6 , now he is triaging issue #6526 and maybe it is caused by this issue

**Comment by jiangxinmeng1** [comment]
This issue can't cause #6256. #6256 panic when new clients. Replay is after it.

**Comment by XuPeng-SH** [comment]
should be fixed by #6460 #6555

**Comment by aressu1985** [comment]
will test this issue tomorrow

**Comment by aressu1985** [comment]
fixed:8f3aa6a337b415b7f99339dcc1f730c8c165ef40
